config/pipeline.yaml

```diff
@@ -1895,6 +1895,19 @@ jobs:
         - media
     kcidb_test_suite: h26forge.debian
 
+  wifi-basic:
```

Comment:
This job definition is not anchored anywhere in the main file - did you encounter any failures in processing scheduler entries with this job definition when it's read from "jobs-chromeos.yaml"?

Reply:
I'm not set up to run the kci tools here, so I'm relying on the CI checks done in github.

My understanding is that the anchors are private to the yaml file. In jobs-chromeos.yaml the only `wifi-basic` entry is in the anchors, so it is not visible from the pipeline.yaml, hence there are no conflicts. I think that's what you were asking, but let me know otherwise.